framework-cloud/framework-openshift/pom.xml

```diff
@@ -42,10 +42,6 @@
     </dependency>
 
     <!-- RH SSO dependencies  -->
-    <dependency>
-      <groupId>org.keycloak</groupId>
-      <artifactId>keycloak-core</artifactId>
-    </dependency>
```

Comment:
This dependency was not necessary as it's already part of keycloak-admin-client.